build-tools/automation/azure-pipelines.yaml

```diff
@@ -996,14 +1033,16 @@ stages:
       job_suffix: One .NET
       nunit_categories: $(DotNetNUnitCategories)
       target_framework: 'net6.0'
+      provisionatorChannel: ${{ parameters.provisionatorChannel }}
 
 - stage: designer_tests
   displayName: Designer Tests
   dependsOn: mac_build
   condition: and(succeeded(), or(eq(variables['RunAllTests'], true), contains(dependencies.mac_build.outputs['mac_build_create_installers.TestConditions.TestAreas'], 'Designer')))
   jobs:
- # Check - "Xamarin.Android (Designer Tests macOS)"
+  # Check - "Xamarin.Android (Designer Tests macOS)"
   - job: designer_integration_mac
+    condition: false #TODO: Enable once test issues are fixed.
```

Comment:
Should we give an example in the commit message what the problem here was?

Reply:
I think this link should hopefully be sufficient https://dev.azure.com/devdiv/DevDiv/_workitems/edit/1469705, I'll also add a note that the team is working to migrate to `dotnet test` which is why macOS is disabled at the moment.